src/pyhf/infer/test_statistics.py

```diff
@@ -30,13 +30,11 @@ def qmu(mu, data, pdf, init_pars, par_bounds):
         Float: The calculated test statistic, :math:`q_{\mu}`
     """
     tensorlib, optimizer = get_backend()
-    mubhathat = optimizer.constrained_bestfit(
-        loglambdav, mu, data, pdf, init_pars, par_bounds
+    mubhathat, fixed_val = fixed_poi_fit(
+        mu, data, pdf, init_pars, par_bounds, return_fval=True
     )
-    muhatbhat = optimizer.unconstrained_bestfit(
-        loglambdav, data, pdf, init_pars, par_bounds
-    )
-    qmu = loglambdav(mubhathat, data, pdf) - loglambdav(muhatbhat, data, pdf)
+    muhatbhat, float_val = fit(data, pdf, init_pars, par_bounds, return_fval=True)
```

Comment:
```suggestion
    muhatbhat, fluid_result = fit(data, pdf, init_pars, par_bounds, return_fit_result=True)
```

Reply:
something that floats is not really flows, no? :)

Reply:
i misinterpreted this as a floating type... `float_val` or `int_val`. `fixed` and `fluid` borrows from web-design (e.g. [fixed vs fluid](https://www.techprevue.com/fixed-layout-vs-fluid-layout-web-design/)).